### PR TITLE
Issue rotheross/otobo#534 migration streamlining

### PR DESCRIPTION
**Pull request by bschmalhofer**
@GritRother-OTOBO  hast du mal einen Blick auf die Migrations-Doku ?

### CODE REVIEW

**Review by GritRother-OTOBO — CHANGES_REQUESTED** [review]
Hi Bernhard, 

I've found a few more things in this file; I'd ask you to change when working on it anyways: (changes highlighted in the screenshot)


> Step 1: Install the new OTOBO system
Warning
"Apache has difficulties with running two independent applications under mod_perl on the same server. Therefore, if you installed OTOBO on the same server as your OTRS / ((OTRS)) Community Edition, disable mod_perl for OTOBO as long as both systems are running. "

=> "Apache has difficulties with running two independent applications under mod_perl on the same server. Therefore, in environments with OTOBO on the same server as OTRS / ((OTRS)) Community Edition, disable mod_perl for OTOBO as long as both systems are running. "


> Step 3a non-Docker: Preparing the OTRS / ((OTRS)) Community Edition system
…
Please login to the OTOBO Admin Area Admin -> System Maintenance and add a new system maintenance slot for a few hours. After that, delete all agent and user sessions (Admin -> Sessions) and logout yourself.

=> Müsste das hier nicht OTRS sein statt OTOBO? 


> Step 6: Known Migration Issues
…
2. Final page of the migration has strange layout due to missing CSS files

This can happen when the setting ScriptAlias has a non-standard value. The migration simple substitutes otrs for otobo. This might lead to the effect that the CSS and JavaScript can no longer be retrieved in OTOBO. When that happens, please check the settings in Kernel/Config.pm and revert them to sane values.

=>
2. Final page of the migration has a strange layout due to missing CSS files

This can happen when the setting ScriptAlias has a non-standard value. The migration simply substitutes otrs for otobo. This might lead to the effect that the CSS and JavaScript can no longer be retrieved in OTOBO. When that happens, please check the settings in Kernel/Config.pm and revert them to sane values.

Thank you!

Bildschirmfoto 2020-10-16 um 13 14 26

File: content/migration-from-otrs-6.rst [hunk 1]
```diff
@@ -24,15 +24,19 @@ With the OTOBO Migration Interface it is possible to perform the following migra
...
+2. A streamlined migration where the source database is first copied to the target database server.
 
-3. It is irrelevant whether your OTRS/ ((OTRS)) Community Edition was previously installed on two separate servers (application and database servers), or whether you want to change OTOBO to such a configuration.
+3. A general migration where many combinations are possible.
 
-4. It is possible to migrate from any of the supported databases to any other one.
+    1. A migration and simultaneous move to a new application server and operating system.
 
-5. It is possible to switch from any supported operating system to any other supported operating system during the migration.
+    2. It is irrelevant whether your OTRS/ ((OTRS)) Community Edition was previously installed on two separate servers (application and database servers), or whether you want to change OTOBO to such a configuration.
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:33]: Noch ein bisschen ausgebaut:
    2. Separate servers: It does not matter whether your OTRS/ ((OTRS)) Community Edition was previously installed on two separate servers (application and database servers). It also does not matter whether you want to change OTOBO to such a configuration.

File: content/migration-from-otrs-6.rst [hunk 2]
```diff
@@ -24,15 +24,19 @@ With the OTOBO Migration Interface it is possible to perform the following migra
...
 
-6. It is possible to migrate to a Docker based installation of OTOBO 10.
+    3. It is possible to migrate from any of the supported databases to any other one.
+
+    4. It is possible to switch from any supported operating system to any other supported operating system during the migration.
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:37]: Done.

File: content/migration-from-otrs-6.rst [hunk 3]
```diff
@@ -24,15 +24,19 @@ With the OTOBO Migration Interface it is possible to perform the following migra
...
+3. A general migration where many combinations are possible.
 
-4. It is possible to migrate from any of the supported databases to any other one.
+    1. A migration and simultaneous move to a new application server and operating system.
 
-5. It is possible to switch from any supported operating system to any other supported operating system during the migration.
+    2. It is irrelevant whether your OTRS/ ((OTRS)) Community Edition was previously installed on two separate servers (application and database servers), or whether you want to change OTOBO to such a configuration.
 
-6. It is possible to migrate to a Docker based installation of OTOBO 10.
+    3. It is possible to migrate from any of the supported databases to any other one.
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:35]: Done

File: content/migration-from-otrs-6.rst [hunk 4]
```diff
@@ -24,15 +24,19 @@ With the OTOBO Migration Interface it is possible to perform the following migra
...
+
+    4. It is possible to switch from any supported operating system to any other supported operating system during the migration.
+
+    5. It is possible to migrate to a Docker based installation of OTOBO 10.
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:39]: Done

File: content/migration-from-otrs-6.rst [hunk 5]
```diff
@@ -24,15 +24,19 @@ With the OTOBO Migration Interface it is possible to perform the following migra
...
 1. A 1:1 migration on the same application server with the same database server.
 
-2. A migration and simultaneous move to a new application server and operating system.
+2. A streamlined migration where the source database is first copied to the target database server.
 
-3. It is irrelevant whether your OTRS/ ((OTRS)) Community Edition was previously installed on two separate servers (application and database servers), or whether you want to change OTOBO to such a configuration.
+3. A general migration where many combinations are possible.
 
-4. It is possible to migrate from any of the supported databases to any other one.
+    1. A migration and simultaneous move to a new application server and operating system.
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:31]: Done

File: content/migration-from-otrs-6.rst [hunk 6]
```diff
@@ -211,25 +196,142 @@ Please make sure there are no running services or cron jobs.
     otrs> /opt/otrs/bin/otrs.Console.pl Maint::WebUploadCache::Cleanup
 
 
+Step 3b Docker: make required data available inside container
+-------------------------------------------------------------------
+
+Some specifities have to be considered when the targeted OTOBO installation runs under Docker,
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:201]: Done

File: content/migration-from-otrs-6.rst [hunk 7]
```diff
@@ -211,25 +196,142 @@ Please make sure there are no running services or cron jobs.
...
+
+The most relevant effect is that processes running in a Docker container generally cannot access directories
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:203]: Done

File: content/migration-from-otrs-6.rst [hunk 8]
```diff
@@ -211,25 +196,142 @@ Please make sure there are no running services or cron jobs.
...
-    In order to make sure that there is a database user who can read the data, it might be worthwhile to create a dedicated user.
-    E.g. ``CREATE USER 'otrs_migration'@'%' IDENTIFIED BY 'otrs_migration'`` and
-    ``GRANT SELECT, SHOW VIEW ON otrs.* TO 'otrs_migration'@'%'``.
+    The default values for OTRS database user and password are taken from *Kernel/Config.pm* in the OTRS home directory.
+    Change the proposed setting if you are working with a database user that is dedicated to the migration.
+    Also change the settings when you work with a database that was copied into the *otobo_db_1* Docker container.
 
+.. note::
+
+    In the Docker case, a database runnung on the Docker host won't be reachable via ``127.0.0.1`` from within the Docker container.
```
[the remark this answers is not in the document]
Comment (GritRother-OTOBO) [reply content/migration-from-otrs-6.rst:322]: @bes - Achtung typo! ;) runnung statt running

File: content/migration-from-otrs-6.rst [hunk 9]
```diff
@@ -211,25 +196,142 @@ Please make sure there are no running services or cron jobs.
...
 
+Step 3b Docker: make required data available inside container
+-------------------------------------------------------------------
+
+Some specifities have to be considered when the targeted OTOBO installation runs under Docker,
+
+The most relevant effect is that processes running in a Docker container generally cannot access directories
+outside the container. There is an exception though: directories mounted as volumes into the container can be accessed.
+
+The other effect is that the MariaDB database running in `otobo_db_1` is not accessible outside the container network.
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:206]: Done

File: content/migration-from-otrs-6.rst [hunk 10]
```diff
@@ -211,25 +196,142 @@ Please make sure there are no running services or cron jobs.
...
+
+.. note::
+
+    In the sample commands we assume that the user **docker_admin** is used for interacting with Docker.A
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:210]: Done.

File: content/migration-from-otrs-6.rst [hunk 11]
```diff
@@ -211,25 +196,142 @@ Please make sure there are no running services or cron jobs.
...
+
+First we need to find out where the volume *otobo_opt_otobo* is available on the Docker host.
+
+.. code-block:: bash
+
+    docker_admin> otobo_opt_otobo_mp=$(docker volume inspect --format '{{ .Mountpoint }}' otobo_opt_otobo)
+    docker_admin> echo $otobo_opt_otobo_mp  # just a sanity check
+
+For safe copying, we use ``rsync``.
+Depending on your Docker setup the command ``rsync`` might need to run with ``sudo``.
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:234]: Done

File: content/migration-from-otrs-6.rst (hunk 8 above)
[the remark this answers is not in the document]
Comment (bes) [reply content/migration-from-otrs-6.rst:322]: I think you tagged the wrong person.

File: content/migration-from-otrs-6.rst [hunk 12]
```diff
@@ -211,25 +196,142 @@ Please make sure there are no running services or cron jobs.
...
+The copied database will be read by the database user *otobo* during the migration. Therefore, *otobo*
+needs to be given read access to the copied database.
+
+.. code-block:: bash
+
+    docker_admin> # note that 'root' and 'otobo' have different passwords
+    docker_admin> docker exec -i otobo_db_1 mysql -u root  -p<root_secrect>       -e "GRANT SELECT, SHOW VIEW, DROP, ALTER ON otrs.* TO 'otobo'@'%'"
+    docker_admin> docker exec -i otobo_db_1 mysql -u otobo -p<otobo_secrect> otrs -e "SELECT COUNT(*), DATABASE(), USER(), NOW() FROM ticket"
+
+When performing the migration with the web-based migration tool, please enter the following values when prompted:
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:285]: Done

File: content/migration-from-otrs-6.rst [hunk 13]
```diff
@@ -211,25 +196,142 @@ Please make sure there are no running services or cron jobs.
...
 Step 4: Perform the Migration!
 ---------------------------------
 
-Please use the web migration tool at http://localhost/otobo/migration.pl (replace "localhost" with your OTOBO hostname)
+Please use the web migration tool at http://localhost/otobo/migration.pl (replace "localhost" with your OTOBO hostname and potentially add the port)
 and follow the process.
 
+.. warning::
+
+    Sometimes a bug pops up where the changed setting for SecureMode is not recognised. In this case restart the webserver.
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:300]: SecureMode kann man aktivieren oder deactivieren. Mit recognised meinte ich dass nicht erkannt wurde, das sich die Einstellung geändert hat. Das ganze ist der Bug https://github.com/RotherOSS/otobo/issues/540.

Ich habe jedenfalls den Abschnitt neu formuliert.

File: content/migration-from-otrs-6.rst [hunk 14]
```diff
@@ -211,25 +196,142 @@ Please make sure there are no running services or cron jobs.
...
 
 .. note::
 
-    In the Docker case, a local database won't be reachable via ``127.0.0.1`` from within the Docker container.
-    Pick one of the IP-addresses reported by ``hostname --all-ip-addresses`` instead for ``OTRS Server``.
-    In order to make sure that there is a database user who can read the data, it might be worthwhile to create a dedicated user.
-    E.g. ``CREATE USER 'otrs_migration'@'%' IDENTIFIED BY 'otrs_migration'`` and
-    ``GRANT SELECT, SHOW VIEW ON otrs.* TO 'otrs_migration'@'%'``.
+    The default values for OTRS database user and password are taken from *Kernel/Config.pm* in the OTRS home directory.
+    Change the proposed setting if you are working with a database user that is dedicated to the migration.
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:317]: Done

File: content/migration-from-otrs-6.rst [hunk 15]
```diff
@@ -211,25 +196,142 @@ Please make sure there are no running services or cron jobs.
...
+    Also change the settings when you work with a database that was copied into the *otobo_db_1* Docker container.
 
+.. note::
+
+    In the Docker case, a database runnung on the Docker host won't be reachable via ``127.0.0.1`` from within the Docker container.
+    This means that the setting ``127.0.0.1`` won't be valid for the input field ``OTRS Server``.
+    In that case, enter for ``OTRS Server`` one of the alternative IP-addresses reported by the command ``hostname --all-ip-addresses``.
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:324]: Done.

File: content/migration-from-otrs-6.rst [hunk 16]
```diff
@@ -211,25 +196,142 @@ Please make sure there are no running services or cron jobs.
...
+
+.. note::
 
+    When migrating to a new application server, or when migration to a Docker-based installation, the database often can't be accessed
+    from the target installation. This is usually due to that the otobo database user can only connect from the same host as the database runs on.
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:329]: Done.

File: content/migration-from-otrs-6.rst [hunk 17]
```diff
@@ -211,25 +196,142 @@ Please make sure there are no running services or cron jobs.
...
+    In order to allow access anyways it is recommended to create a dedicated database user for the migration.
+    E.g. ``CREATE USER 'otrs_migration'@'%' IDENTIFIED BY 'otrs_migration';`` and
+    ``GRANT SELECT, SHOW VIEW ON otrs.* TO 'otrs_migration'@'%';``.
+    After the migration this user can be dropped again: ``DROP USER 'otrs_migration'@'%';``.
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:333]: Done.

File: content/migration-from-otrs-6.rst [hunk 18]
```diff
@@ -211,25 +196,142 @@ Please make sure there are no running services or cron jobs.
...
+Optionally copy the otrs database schema to the containerised database server
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The migration can take copy databases more effectively when soure and target are located on
+the same database server. In order to take advantage of that we can optionally import the source
+database into the database server runnning in the *otobo_db_1* container.
+This approach allow to simply rename tables instead of copying the rows.
+
+First we need a dump of the source OTRS database. The dumping can, but doesn't have to, be performed on the Docker host.
+Here we concentrate on the case, where OTRS is running with MySQL on the Docker host and uses the database
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:257]: Done.

File: content/migration-from-otrs-6.rst [hunk 19]
```diff
@@ -211,25 +196,142 @@ Please make sure there are no running services or cron jobs.
...
+**otrs**.
+
+The database can be dumped with the command ``mysqldump``
+
+.. code-block:: bash
+
+    docker_admin> mysqldump -h localhost -u root -p --databases otrs --dump-date > mysqldump_otrs.sql
+    docker_admin> head -n 30 mysqldump_otrs.sql # just a sanity check
+
+For importing the dumped database we run ``mysql`` inside the running Docker container *otobo_db_1*.
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:267]: Done.

File: content/migration-from-otrs-6.rst [hunk 20]
```diff
@@ -211,25 +196,142 @@ Please make sure there are no running services or cron jobs.
...
+
+Optionally copy the otrs database schema to the containerised database server
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The migration can take copy databases more effectively when soure and target are located on
+the same database server. In order to take advantage of that we can optionally import the source
+database into the database server runnning in the *otobo_db_1* container.
+This approach allow to simply rename tables instead of copying the rows.
+
+First we need a dump of the source OTRS database. The dumping can, but doesn't have to, be performed on the Docker host.
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:256]: Ich habe den Abschnitt neu formulliert.

File: content/migration-from-otrs-6.rst (hunk 8 above)
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:322]: Done.

File: content/migration-from-otrs-6.rst [hunk 21]
```diff
@@ -211,25 +196,142 @@ Please make sure there are no running services or cron jobs.
...
 
+.. note::
+
+    In the Docker case, a database runnung on the Docker host won't be reachable via ``127.0.0.1`` from within the Docker container.
+    This means that the setting ``127.0.0.1`` won't be valid for the input field ``OTRS Server``.
+    In that case, enter for ``OTRS Server`` one of the alternative IP-addresses reported by the command ``hostname --all-ip-addresses``.
+
+.. note::
 
+    When migrating to a new application server, or when migration to a Docker-based installation, the database often can't be accessed
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:328]: Done

**Review by GritRother-OTOBO — CHANGES_REQUESTED** [review]
Proofread and added a few changes.

File: content/migration-from-otrs-6.rst [hunk 22]
```diff
@@ -248,23 +246,31 @@ This copied directory will be available as */opt/otobo/var/tmp/copied_otrs* with
 Optionally copy the otrs database schema to the containerised database server
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The migration can take copy databases more effectively when soure and target are located on
-the same database server. In order to take advantage of that we can optionally import the source
-database into the database server runnning in the *otobo_db_1* container.
-This approach allow to simply rename tables instead of copying the rows.
+In the general case, the data in the database table is copied row for row from the OTRS database
+into the OTOBO database. This approach is time consuming and can be optimised.
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:250]: Done.

File: content/migration-from-otrs-6.rst [hunk 23]
```diff
@@ -248,23 +246,31 @@ This copied directory will be available as */opt/otobo/var/tmp/copied_otrs* with
 Optionally copy the otrs database schema to the containerised database server
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The migration can take copy databases more effectively when soure and target are located on
-the same database server. In order to take advantage of that we can optionally import the source
-database into the database server runnning in the *otobo_db_1* container.
-This approach allow to simply rename tables instead of copying the rows.
+In the general case, the data in the database table is copied row for row from the OTRS database
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:249]: Done.

File: content/migration-from-otrs-6.rst [hunk 24]
```diff
@@ -248,23 +246,31 @@ This copied directory will be available as */opt/otobo/var/tmp/copied_otrs* with
...
+into the OTOBO database. This approach is time consuming and can be optimised.
+For enabling a performance optimisation, we first to create a temporary copy of the OTRS database
+in the same dabase server as is used for OTOBO.
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:252]: Done.

File: content/migration-from-otrs-6.rst [hunk 25]
```diff
@@ -248,23 +246,31 @@ This copied directory will be available as */opt/otobo/var/tmp/copied_otrs* with
...
+In our case, this is the MariaDB-server running in the container ``otobo_db_1``.
+After creating the temporary copy, the relevant OTRS tables can then be moved into the OTOBO database.
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:254]: Done.

File: content/migration-from-otrs-6.rst [hunk 26]
```diff
@@ -248,23 +246,31 @@ This copied directory will be available as */opt/otobo/var/tmp/copied_otrs* with
...
+
+.. warning::
+
+    Moving tables from one schema to another schema makes the source schema unusable.
+    So make sure that OTRS database is really a throwaway copy of the productive OTRS database.
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:259]: Done.

File: content/migration-from-otrs-6.rst [hunk 27]
```diff
@@ -248,23 +246,31 @@ This copied directory will be available as */opt/otobo/var/tmp/copied_otrs* with
...
+In the general case, the data in the database table is copied row for row from the OTRS database
+into the OTOBO database. This approach is time consuming and can be optimised.
+For enabling a performance optimisation, we first to create a temporary copy of the OTRS database
+in the same dabase server as is used for OTOBO.
+In our case, this is the MariaDB-server running in the container ``otobo_db_1``.
+After creating the temporary copy, the relevant OTRS tables can then be moved into the OTOBO database.
+
+.. warning::
+
+    Moving tables from one schema to another schema makes the source schema unusable.
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:258]: Done.

File: content/migration-from-otrs-6.rst [hunk 28]
```diff
@@ -248,23 +246,31 @@ This copied directory will be available as */opt/otobo/var/tmp/copied_otrs* with
...
+    So make sure that OTRS database is really a throwaway copy of the productive OTRS database.
 
-First we need a dump of the source OTRS database. The dumping can, but doesn't have to, be performed on the Docker host.
-Here we concentrate on the case, where OTRS is running with MySQL on the Docker host and uses the database
-**otrs**.
+First of all, we need a dump of the source OTRS database.
+When ``mysqldump`` is installed and a database connection is possible,
+then the database dump can be created on the Docker host.
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:263]: Done.

File: content/migration-from-otrs-6.rst [hunk 29]
```diff
@@ -198,16 +198,14 @@ Please make sure there are no running services or cron jobs.
 Step 3b Docker: make required data available inside container
 -------------------------------------------------------------------
 
-Some specifities have to be considered when the targeted OTOBO installation runs under Docker,
-
-The most relevant effect is that processes running in a Docker container generally cannot access directories
+There are some specifics to be considered when your OTOBO installation is running under Docker.
+Most relevant is that processes running in a Docker container generally cannot access directories
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:202]: Done.

File: content/migration-from-otrs-6.rst [hunk 30]
```diff
@@ -248,23 +246,31 @@ This copied directory will be available as */opt/otobo/var/tmp/copied_otrs* with
...
+    Moving tables from one schema to another schema makes the source schema unusable.
+    So make sure that OTRS database is really a throwaway copy of the productive OTRS database.
 
-First we need a dump of the source OTRS database. The dumping can, but doesn't have to, be performed on the Docker host.
-Here we concentrate on the case, where OTRS is running with MySQL on the Docker host and uses the database
-**otrs**.
+First of all, we need a dump of the source OTRS database.
+When ``mysqldump`` is installed and a database connection is possible,
+then the database dump can be created on the Docker host.
+Alternatively the database can be dumped on another server and then be transfered to the Docker host.
```
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:264]: Done.

File: content/migration-from-otrs-6.rst [hunk 31]
```diff
@@ -78,7 +78,7 @@ We strongly recommend to read the :doc:`installation` chapter.
 
     Under Apache, there are pitfalls with running two independent applications under mod_perl on the same server.
     These pitfalls can be alleviated with the mod_perl setting ``PerlOptions +Parent``. This setting makes sure
-    that the relevant application uses it's own dedicated Perl interpreter. Please check your Apache config files in
+    that the applications uses their own dedicated Perl interpreter. Please check your Apache config files in
```
Comment (GritRother-OTOBO) [inline content/migration-from-otrs-6.rst:81]: @bes - da steht noch useS
=> ... that the applications use their own dedicated Perl interpreter. Please check your Apache config files in

Der Teufel ist ein Eichhörnchen ;)

**Review by GritRother-OTOBO — CHANGES_REQUESTED** [review]
Sorry ....

File: content/migration-from-otrs-6.rst [hunk 32]
```diff
@@ -286,8 +286,14 @@ Note that the password for the database root is now the password that has been s
 
     docker_admin> docker exec -i otobo_db_1 mysql -u root -p<root_secret> < otrs_schema.sql
     docker_admin> docker exec -i otobo_db_1 mysql -u root -p<root_secret> < otrs_data.sql
-    docker_admin> docker exec -i otobo_db_1 mysql -u root -p<root_secret> -e 'SHOW DATABASES'     # sanity check
-    docker_admin> docker exec -i otobo_db_1 mysql -u root -p<root_secret> otrs -e 'SHOW TABLES'   # sanity check
+
+For a quick checl whether the import worked, you can run the following commands.
```
Comment (GritRother-OTOBO) [inline content/migration-from-otrs-6.rst:290]: For a quick check whether the import worked, you can run the following commands:

**Review by GritRother-OTOBO — CHANGES_REQUESTED** [review]
checl => check

File: content/migration-from-otrs-6.rst [hunk 33]
```diff
@@ -28,9 +28,9 @@ With the OTOBO Migration Interface it is possible to perform the following migra
 
 3. A general migration where many combinations are possible.
 
-    1. Change server and OS: Migrate and simultaneously move to a new application server and operating system.
+    1. Change server: Migrate and simultaneously move to a new application server.
 
-    2. Separate servers: It does not matter whether your OTRS/ ((OTRS)) Community Edition was previously installed on two separate servers (application and database servers). It also does not matter whether you want to change OTOBO to such a configuration.
+    2. Separate servers: It does not matter whether your OTRS/ ((OTRS)) Community Edition was previously installed on two separate servers (application and database server). It also does not matter whether you want to change OTOBO to such a configuration.
```
Comment (GritRother-OTOBO) [inline content/migration-from-otrs-6.rst:33]: Bin wieder drüber gestolpert. Vielleicht so?
Separate servers: Regardless of the prior setup of your OTRS / ((OTRS)) Community Edition on one or two servers (application and database server), you are free to change this configuration for OTOBO.

File: content/migration-from-otrs-6.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst]: Looks like this stuff has been covered.

File: content/migration-from-otrs-6.rst (hunk 31 above)
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst:81]: Leider wahr.

File: content/migration-from-otrs-6.rst (hunk 33 above)
Comment (bschmalhofer) [reply content/migration-from-otrs-6.rst]: Ich habe es neu, etwas positiver, formuliert,

**Pull request merged**